libs/content/workspace/src/tab/markdown_editor/editor.rs

```diff
@@ -18,79 +18,50 @@ use crate::tab::markdown_editor::input::click_checker::{ClickChecker, EditorClic
 use crate::tab::markdown_editor::input::cursor::PointerState;
 use crate::tab::markdown_editor::input::events;
 use crate::tab::markdown_editor::offset_types::{DocCharOffset, RangeExt as _};
-use crate::tab::markdown_editor::style::{BlockNode, InlineNode, ListItem, MarkdownNode};
 use crate::tab::markdown_editor::{ast, bounds, galleys, images, register_fonts};
 use crate::tab::EventManager as _;
 
 #[derive(Debug, Serialize, Default)]
 pub struct EditorResponse {
+    // state changes
     pub text_updated: bool,
-    pub suggested_rename: Option<String>,
-
-    pub scroll_updated: bool,
-
-    pub show_edit_menu: bool,
```

Comment:
I don't know how we decide where and when to show edit menu anymore but apparently it's not using this

Reply:
We need `show_edit_menu`, `edit_menu_x` and `edit_menu_y` for android. Its how we know when to show the context menu (cut, copy, paste, etc).

Reply:
Note, the PR where android adds this context menu feature and thus needs those fields) is unmerged as of right now. Although it will be merged soon.